OneCommunityGlobal/HGNRest
# Carl-fix-suggestion-icon-modal

# Description
The "Suggestion Icon Modal" stopped over a week ago. Initially, when the Suggestion Icon is clicked a modal appears on the screen allowing suggestions to be made.

## Related PRS (if any):
This backend PR is related to the [#1292](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1292) frontend PR.
To test this backend PR you need to checkout the [#1292](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1292)  frontend PR.
…

## Main changes explained:
- Added error handling to the functions for effective debugging
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ Suggestion Icon…
5. verify function “A” (feel free to include screenshot here)
